README.md

```diff
@@ -1,7 +1,151 @@
-
 # object-store-client
 
-This is a placeholder README.md for a new repository
+object-store-client is a Scala client to interact with the [object-store](https://github.com/hmrc/object-store) service. 
+The methods available on the client encapsulates the HTTP interaction with object-store service which makes it easier for the consuming service to interact with object-store service.
+
+The operations available via object-store-client are 
+- Put a new document
+- Download a document
+- Delete a document
+- List documents under a path
+
+Read more [here](https://github.com/hmrc/object-store/blob/master/README.md) about object-store.
+
+We also have a working example [hello-world-object-store](https://github.com/hmrc/hello-world-object-store) which shows in detail on how to use the object-store-client.
+
+## Using object-store-client
+    
+### Add object-store-client dependency
+
+In the SBT build add:
+
+```sbt
+resolvers += Resolver.bintrayRepo("hmrc", "releases")
+
+libraryDependencies += "uk.gov.hmrc.objectstore" %% "object-store-client-play-xx" % "x.x.x"
+```
+
+### Configuration
+
+- Ensure `appName` is configured in `application.conf`
+
+
+- Enable `uk.gov.hmrc.objectstore.client.play.modules.ObjectStoreModule` in the `application.conf` by adding
+```hocon
+play.modules.enabled += "uk.gov.hmrc.objectstore.client.play.modules.ObjectStoreModule"
+```
+
+- Configure object-store HTTP configuration under `microservice.services` block in `application.conf`
+```hocon
+object-store {
+  host = localhost
+  port = 8464
+}
+```
+
+Also, override this configuration in app-config-base so this service could interact with object-store service in deployed environments. 
+
+- Configure Authorization token in `application.conf` which grants permissions to this service to interact with object-store via object-store-client.
```

Comment:
"Configure Authorization token in `application.conf` which grants" ->
"Configure **the** Authorization token in `application.conf` which grants"